### ISSUE TITLE
Extend the support for logo in Template Library

### ISSUE THREAD

**Issue by algomaster99** [accordproject/cicero-ui#364]
**Is your feature request related to a problem? Please describe.**
We can add the support for the logo in the `TemplateLibrary` Component since https://github.com/accordproject/cicero/pull/533 is merged.

**Describe the solution you'd like**
I am actually not sure how we can extend the support here :sweat: @irmerk @DianaLease I would need your suggestions. :sweat_smile:

**Comment by irmerk** [comment]
Yes this is a good issue. It will wait for https://github.com/accordproject/cicero-ui/issues/357, but should be _relatively_ straightforward to swap out the current logo being an image with the `src` populated by props.

**Comment by irmerk** [comment]
@algomaster99 #357 is done and is no longer a blocker.
